x/autopilot/types/parser.go

```diff
@@ -43,7 +45,10 @@ func (m StakeibcPacketMetadata) Validate() error {
 	if err != nil {
 		return err
 	}
-	if m.Action != "LiquidStake" {
+	switch m.Action {
+	case "LiquidStake":
+	case "RedeemStake":
```

Comment:
these seem that are not set up

Reply:
It's validation if the action is one of LiquidStake or RedeemStake, more could be added later and just used this format for validation